ui/packages/tidb-dashboard-lib/src/apps/SystemReport/context/index.ts

```diff
@@ -29,8 +29,13 @@ export interface ISystemReportDataSource {
   ): AxiosPromise<DiagnoseReport>
 }
 
+export interface ISystemReportEvent {
+  getFullReportLink(reportID: string): string
+}
+
 export interface ISystemReportContext {
   ds: ISystemReportDataSource
+  event: ISystemReportEvent
   cfg: IContextConfig & { publicPathBase: string }
```

Comment:
How about put the `getFullReportLink ` into cfg?

Reply:
Good idea!